### DISCUSSION TITLE
As shown in the figure, do not slide up and down

### DISCUSSION THREAD

**Discussion by 183868hlj** [violentmonkey/violentmonkey#2192] [Q&A] [answered]
As shown in the figure, do not slide up and down
![Screenshot_2024-09-21-04-14-52-29_1526bbe67bba6d33b86f0f3237fecb5d](https://github.com/user-attachments/assets/1ff5eae2-2865-4bf9-bb60-d43644348d1e)

**Comment by tophf** [comment, marked as answer]
This will be fixed in 2.25.0. Until then the workaround is to go to Violentmonkey settings and set `Custom CSS` to this:
```css
.touch .page-popup { max-height: 100vh }
```

**Reply by 183868hlj** [reply]
谢谢！
